cognite_toolkit/cdf_tk/load.py

```diff
@@ -855,12 +863,46 @@ def create(self, items: Sequence[T_Resource], drop: bool, filepath: Path) -> T_R
                         if item.external_id == ext_id:
                             items.remove(item)
                 try:
-                    return ExtractionPipelineList(self.client.extraction_pipelines.create(items))
+                    extractionPipelineList = ExtractionPipelineList(self.client.extraction_pipelines.create(items))
                 except Exception as e:
                     print(f"[bold red]ERROR:[/] Failed to create extraction pipelines.\n{e}")
                     self.ToolGlobals.failed = True
                     return None
-            return None
+
+        file_name = filepath.stem.split(".", 2)[1]
```

Comment:
This is just to remove the number at the beginning?
I think this is more robust
```suggestion
        file_name =re.sub(r'^(\d+)\.', "", filepath.stem)
```